src/Symfony/Component/HttpKernel/Attribute/Serialize.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Controller tag to serialize response.
+ */
+#[\Attribute(\Attribute::TARGET_METHOD)]
+class Serialize
```

Comment:
`AsSerializedResponse` would also work :thinking:

Reply:
indeed, others attribute are prefixed with `As` when they describe more the config of the class (AsCommand or AsMessageHandler)
here it is like a mapping/transformation, perhaps like your other PR https://github.com/symfony/symfony/pull/49138 `MapResponse`